src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -3238,98 +3254,5 @@ private static StandardCredentials lookupCredentials(Item context, String creden
                         CredentialsMatchers.withId(credentialsId));
     }
 
-    public static class AdditionalCredentials extends AbstractDescribableImpl<AdditionalCredentials> {
```

Comment:
No, you must leave this class where it was for reasons of settings compatibility. Nothing is stopping `SubversionSCMSource` from reusing it here.